spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -2746,6 +2746,7 @@ def keyReleaseEvent(self, event):
         """Override Qt method."""
         self.timer_syntax_highlight.start()
         super(CodeEditor, self).keyReleaseEvent(event)
+        event.ignore()

```

Comment:
This was the error, In qt4 the event wasn't propagated to the parent (I had tested it with pyqt5)